test/test_es_utils.py

```diff
@@ -104,10 +105,12 @@ def es_client_fixture(integrated_ff):
     return create_es_client(integrated_ff['es_url'])
 
 
+@pytest.mark.skip(reason="Direct ES search cannot work outside the firewall.")
 @pytest.mark.integrated
+@pytest.mark.direct_es_query
 def test_lucene_query_basic(es_client_fixture):
     """ Tests basic lucene queries via the underlying endpoint on mastertest """
-    results = execute_lucene_query_on_es(client=es_client_fixture, index='fourfront-mastertestuser', query={})
+    results = execute_lucene_query_on_es(client=es_client_fixture, index=f'{INTEGRATED_ENV_INDEX}user', query={})
```

Comment:
I see the use now - why do it this way though? You can get this info from the health page. The variable name is also not good... this is an ES index namespace.